packages/acceptit-Core.package/ACUserStory.class/class/means.st

```diff
@@ -1,4 +1,3 @@
-accessing
+constants
 means
-
-	^means
\ No newline at end of file
+	"self subclassResponsibility."
```

Comment:
If you checkout this branch locally, you can try to uncomment this stuff. But in my experience the self subclass responsibility throws errors while testing because means is implemented later than when the class gets created. Which results in exceptions and failing tests despite everything being created correctly.